test/src/tests-batchUploader.ts

```diff
@@ -88,13 +204,15 @@ describe('batch uploader', () => {
                 uploader.queueEvent(({} as unknown) as SDKEvent);
 
                 expect(uploader.pendingEvents).to.eql([]);
-                expect(uploader.pendingUploads).to.eql([]);
+                // TODO: Should we remove because of Vault?
+                // expect(uploader.pendingUploads).to.eql([]);
             });
         });
     });
 
     describe('batching via window.fetch', () => {
         beforeEach(function() {
+            window.fetchMock.restore();
```

Comment:
Nit - this is probably fine as is, but technically if should just be in `afterEach` and maybe once in a `before` block.